### PR TITLE
Keep JSBI bundle seperated and consistent

### PR DESCRIPTION
**Pull request by Leslie-Wong-H**
Related issue: #79.

Thanks for the PR invitation. @12wrigja

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "include": ["lib/"],
```
Comment (12wrigja) [inline tsconfig.json:2]: What is this for?

**Review by 12wrigja — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (Leslie-Wong-H) [reply tsconfig.json:2]: Explicitly include which ts file to compile. By default tsconfig.json will search all the ts files within the root directory to compile, excluding those specified by "exclude". 

> https://www.typescriptlang.org/tsconfig#include

Comment (12wrigja) [reply tsconfig.json:2]: Gotcha. I don't see any harm either way, so let's just leave it as-is.

**Comment by 12wrigja** [comment]
FYI I pulled down your changes and ran the resulting UMD bundle against IE11 and it seems to work just fine. Thanks for fixing this mess :sweat_smile:

**Comment by Leslie-Wong-H** [comment]
Closes #79.

**Pull request merged**